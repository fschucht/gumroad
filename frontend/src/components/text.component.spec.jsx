import renderer from 'react-test-renderer';
import { Text } from './text.component';

describe('Text', () => {
  it('should render a Text component', () => {
    const renderedComponent = renderer.create(<Text>Hello World!</Text>).toJSON()

    expect(renderedComponent).toMatchSnapshot()
  })
})