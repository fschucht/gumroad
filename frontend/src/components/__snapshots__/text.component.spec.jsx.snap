// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text should render a Text component 1`] = `
<p
  className="text"
>
  Hello World!
</p>
`;
